evergreen/install_baas.sh

```diff
@@ -242,6 +254,15 @@ if [ ! -x "$WORK_PATH/mongodb-binaries/bin/mongod" ]; then
     chmod +x ./mongodb-binaries/bin/*
 fi
 
+if [[ -n "$MONGOSH_DOWNLOAD_URL" ]] && [[ ! -x "$WORK_PATH/mongodb-binaries/bin/mongosh" ]]; then
+    echo "Downloading mongosh"
+    $CURL -sLS $MONGOSH_DOWNLOAD_URL --output mongosh-binaries.zip
+    unzip -jnqq mongosh-binaries.zip '*/bin/*' -d mongodb-binaries/bin/
+    rm mongosh-binaries.zip
+fi
+
+[[ -n "$MONGOSH_DOWNLOAD_URL" ]] && MONGOSH="mongosh" || MONGOSH="mongo"
+
```

Comment:
Does it matter that there are two different values for the mongod pid file? Was the latter a carry over from an older version of the `install_baas.sh` script or an artifact of running mongod manually?
- `$WORK_PATH/mongod.pid`
- `$WORK_PATH/mongodb-path/mongod.pid`

Reply:
I think that's just an oversight. We can remove line 304 since it's effectively a noop.